crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -421,6 +422,47 @@ impl EventTimelineItem {
             kind,
         }
     }
+
+    /// Gives the information needed to reply to the event of the item.
+    pub fn get_replied_to_info(&self) -> Result<RepliedToInfo, UnsupportedReplyItem> {
```

Comment:
nit: let's get rid of the `get_` prefix everywhere (it's not very idiomatic in Rust)